pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -273,6 +273,21 @@ def test_to_json(self):
         expected = OrderedDict([('schema', schema), ('data', data)])
         assert result == expected
 
+    def test_to_json_orient(self):
+        s1 = pd.Series([1, 2, 3], name='a')
```

Comment:
Can you parametrize these tests?

Reply:
yes, i was not very familiar with pytest.mark.parametrize, feel nice to learn!! and i am not sure if i used it correctly, please let me know how to make it better! thanks!